assets/src/dashboard/app/views/editorSettings/googleAnalytics/index.js

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import {
+  SettingForm,
+  SettingLabel,
+  TextInput,
+  TextInputHelperText,
+} from '../components';
+
+const TEXT = {
+  context: __(
+    "The story editor will append a default, configurable AMP analytics configuration to your story. If you're interested in going beyond what the default configuration is, read this article.",
+    'web-stories'
+  ),
+  label: __('Google Analytics Tracking ID', 'web-stories'),
+  placeholder: __('Enter your Google Analtyics Tracking ID', 'web-stories'),
```

Comment:
For accessibility reasons, it's important that the placeholder is actually a placeholder like `UA-1234578-9` and not an instruction.

Reply:
good call. I was thinking that the label "google analytics tracking id" was plenty but you're right that users of all sorts should have just as much clarity of the form. I've added a visually hidden span that can be set to whatever text we want should we change the placeholder in the future and tied it to the input as the aria-describedBy. 

![Screen Shot 2020-07-02 at 2 15 26 PM](https://user-images.githubusercontent.com/10720454/86410080-ff11c300-bc6e-11ea-9d72-f1b261ec057d.png)